internal/graphql/resolver/kubernetes.go

```diff
@@ -282,3 +284,46 @@ func (r *Resolver) getKubectlDescribe(ctx context.Context, name string, kind str
 		Describe: &details,
 	}, nil
 }
+
+func (r *Resolver) subscribeK8sContexts(ctx context.Context, provider models.Provider, selector model.PageFilter) (<-chan *model.K8sContextsPage, error) {
+	ch := make(chan struct{}, 1)
+	ch <- struct{}{}
```

Comment:
@MUzairS15  Why passing this struct in the channel at this point?

Reply:
So that when a subscription is started for the first time, all the contexts are returned to the client (otherwise for initial, fetch REST API request has to be made)

Reply:
Ok makes sense